Formula/mono.rb

```diff
@@ -86,6 +102,18 @@ class Mono < Formula
   end
 
   def install
+    # Replace hardcoded /usr/share directory. Paths like /usr/share/.mono,
+    # /usr/share/.isolatedstorage, and /usr/share/template are referenced in code.
+    inreplace_files = %w[
+      external/corefx/src/System.Runtime.Extensions/src/System/Environment.Unix.cs
+      mcs/class/corlib/System/Environment.cs
+      mcs/class/corlib/System/Environment.iOS.cs
+      man/mono-configuration-crypto.1
+      man/mono.1
+      man/mozroots.1
+    ]
+    inreplace inreplace_files, %r{/usr/share(?=[/"])}, pkgshare
```

Comment:
Still need to figure out where to store this. I set `pkgshare` as original directory was `/usr/share`. An alternative is `pkgetc` if any data needs to be retained between upgrades.

Reply:
I think if they were originally in `share`, they belong in `pkgshare`.  Typically things in `etc` are what the user may wish to configure, while things in `share` are not supposed to be modified.  So it would be okay for those to be replaced on upgrade.